2022-10-19-recommenders-1.md

```diff
@@ -26,48 +26,115 @@ alternatives instead
 
 This is a proposed plan only, you don’t have to follow it day-by-day.
 
+### Day 1 (19 October, Wednesday)
+
+* Come up with a project idea.
+* Select the dataset for your project.
+* Create a GitHub project.
+* Share your progress in Slack and in social media.
+
+
+### Day 2 (20 October, Thursday)
+
+* Learn the basics about recommendation systems (see Suggested materials).
+* Perform exploratory data analysis of your data in a Jupyter notebook
+* Commit your changes.
+* Share your progress in Slack and in social media.
+
+Suggested materials
+
+* 🗒️ [Background section of Google Recommendation systems course](https://developers.google.com/machine-learning/recommendation)
+* 🏫 [ML Zoomcamp - EDA section](https://github.com/alexeygrigorev/mlbookcamp-code/blob/master/course-zoomcamp/02-regression/03-eda.md)
+
+Found good materials? Create a PR with links!
+
+### Day 3 (21 October, Friday)
+
+* Continue learning the basics about recommendation systems (see Suggested materials).
+* Perform cleaning of your data in a Jupyter notebook.
+* Commit your changes.
+* Share your progress in Slack and in social media.
+
+Suggested materials
+
+* 🗒️ [Background section of Google Recommendation systems course](https://developers.google.com/machine-learning/recommendation)
 
-### Day X (DD MMMM, Day of the week)
 
-* Action 1 
-* Action 2
-* Action 3
-* Create a GitHub project (if day 1)
-* Commit your changes (if day 2+)
-* Share your progress in Slack and in social media
+Found good materials? Create a PR with links!
+
+### Day 4 (22 October, Saturday)
+
+* Learn about Content-based recommendation systems.
```

Comment:
For content-based we wanted to do another week?

Reply:
I think we might want to introduce both during the same week and then get deeper on comparison during the 2nd one.